### PR TITLE
New post notifications

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: lib/freshness.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 var checkFreshness = function (newPosts, cachedPosts) {
   var difference = _.differenceBy(newPosts, cachedPosts, 'id')
-  return difference.length !== 0
+  if (difference.length === 0) return false
```
Comment (levity) [inline lib/freshness.js]: why not just return 0?

**Comment by robbiecarlton** [comment]
done

**Comment by levity** [comment]
:+1:

**Pull request merged**